cmdopt/src/main/scala/pravda/cmdopt/Show.scala

```diff
@@ -0,0 +1,8 @@
+package pravda.cmdopt
+
+trait Show[A, O] {
```

Comment:
Do you really need this type-class? It seems you override `show` method everytime.

Reply:
Yes, we need to be able to `show` the same CommandLine configuration in different modes markdown or console, depending on what instance is available in scope.

Reply:
Hmm, I just can't find usage of it in the code.

Reply:
You may want to look at typeclass instances and their usage.